services/housetables/src/main/java/com/linkedin/openhouse/housetables/services/UserTablesServiceImpl.java

```diff
@@ -96,4 +106,67 @@ public void deleteUserTable(String databaseId, String tableId) {
     htsJdbcRepository.deleteById(
         UserTableRowPrimaryKey.builder().databaseId(databaseId).tableId(tableId).build());
   }
+
+  private List<UserTableDto> listDatabases() {
+    return StreamSupport.stream(htsJdbcRepository.findAllDistinctDatabaseIds().spliterator(), false)
```

Comment:
given we already have metric object installed, shall we also time this method ? 
Not blocking.

Reply:
Sure.